doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
+        ``Admin`` in its own database (:ref:`code_db`). You can use the tool
+        ``pi-manage`` to
+        manage those admins from the command line as the system's root user. (see
+        :ref:`installation`)
+
+        These admin users can logon to the WebUI using the admin's user name and the
+        specified password.
+        These admins are used to get a simple quick start.
+
+        Then you can define realms (see :ref:`realms`), that should be administrative
+        realms. I.e. each user in this realm will have administrative rights in the
+        WebUI.
+
+        .. note:: You need to configure these realms within privacyIDEA. Only
+           after these realms exist, you can raise their rights to an administrative
+           role.
+
+        .. note:: Use this carefully. Imagine you defined a resolver to a specific
+           group in your Active Directory to be the pricacyIDEA admins. Then the Active
+           Directory domain admins can
+           simply add users to be administrator in privacyIDEA.
+
+        You define the administrative realms in the config file ``pi.cfg``, which is
+        usually located at ``/etc/privacyidea/pi.cfg``::
+
+           SUPERUSER_REALM = ["adminrealm1", "super", "boss"]
+
+        In this case all the users in the realms "adminrealm1", "super" and "boss"
+        will have administrative rights in the WebUI, when they login with this realm.
+
+        As for all other users, you can use the :ref:`policy_login_mode` to define,
+        if these administrators should login to the WebUI with their userstore password
+        or with an OTP token.
+
+   Userstore
+        Are the locations, where the users are managed. This can be LDAP directories
+        or especially Active Directory, some times FreeIPA or the Redhat 389 service.
+        But classically users are also located in files like /etc/passwd on standalone unix systems.
+        Web services often use SQL databases as user store.
+
+   Userinfo
+        These are the user attributes as they are determined by the respective resolver.
+        This is configured via the attribute mappings of resolvers (see :ref:`useridresolvers`).
+
+   Challenge
+   Multi Challenge
+        If a user wants to authenticate with his username and password, privacyIDEA will check whether an active
+        challenge response token exists for this user. In this case, the challenge is triggered and privacyIDEA expects a response.
+        If the user now gives the answer expected from the server, the response is accepted and the authentication is successful.
+
+        Multi Challenge is basically a chain of challenges. If, for example, a PIN is stored for a challenge response token, after
+        the first successful challenge, a second one is triggered, in which the PIN is expected as an answer.
+        If the PIN matches, the second challenge is also successful and the user is successfully authenticated.
```

Comment:
Unfortunately this is wong! The PIN triggers the first challenge.

Multichallenge can be used to reset a PIN or with the four-eyes-token.

Reply:
done in f9f28db97f886ae132df666cc409008769c9b311.